packages/fixtures/src/fields.ts

```diff
@@ -0,0 +1,49 @@
+class Fields {
```

Comment:
Not my favorite implementation. Best I could come up with. Suggestions welcomed.

Reply:
Aiming for a way to `reset` and export fields as types. Ideally you'd also be able to dereference them at the top of a fixture file... but I don't think getters can do that.